aim/web/ui/src/components/LineChart/LineChart.scss

```diff
@@ -125,13 +125,31 @@
   .yAxis, .xAxis {
     .tick {
       text {
+        display: none;
+      }
+      .yAxisTick, .xAxisTick {
         color: $text-color;
       }
     }
     .domain {
       stroke: $text-color;
       stroke-width: 0.4;
     }
+    .xAxisTick {
+      text-shadow: 0 1px 0 #fff, 1px 0 0 #fff, 0 -1px 0 #fff, -1px 0 0 #fff;
```

Comment:
please change px to rem by wrapping with toRem method

Reply:
We need to decide with UI team when exactly we use px or toRem(px)
@roubkar what do you think about this ?

Reply:
`px` is good I think, but probably we can have a var for text-shadow?

Reply:
Good idea @roubkar jan will add scss variable for text-shadows

Reply:
interesting i copied the styles from Highplot , will change there too

Reply:
@VkoHov where did you get this text-shadow styles  in HighPlot, i didn’t check it, does it necessary to have ?

Reply:
Could hardly remember this part. I think we can remove this line of code.

Reply:
I think this is unnecessary styles, i will remove them